### PR TITLE
Update LAB to Braze Learning

### PR DESCRIPTION
**Pull request by bre-fitzgerald**
CE rebranded "Learning at Braze (LAB)" to "Braze Learning". This PR updates relevant mentions of LAB and changes lab.braze.com URLs to learning.braze.com to cut down on redirects needed.

### CODE REVIEW

File: _docs/_user_guide/engagement_tools/campaigns/ideas_and_strategies/capturing_lapsing_users.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ tool:
 
 If your audience is dwindling, it’s crucial to try wooing them back. Recognizing this necessity, Braze makes it easy to set up automated recurring re-engagement campaigns to capture lapsing users. You can choose the re-engagement timeframe and recurrence that best suits your app, but to demonstrate, we'll get started with a 14 day re-engagement plan.
 
-For more on targeting users, check out our [LAB course](http://lab.braze.com/campaign-setup-delivery-targeting-conversions) on campaign setup!
+For more on targeting users, check out our [Braze Learning course](http://learning.braze.com/campaign-setup-delivery-targeting-conversions) on campaign setup!
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/campaigns/ideas_and_strategies/capturing_lapsing_users.md:19]:
```suggestion
For more on targeting users, check out our [Braze Learning course](https://learning.braze.com/campaign-setup-delivery-targeting-conversions) on campaign setup!
```

File: _docs/_user_guide/engagement_tools/campaigns/ideas_and_strategies/active_user_campaigns.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ tool:
 
 Reaching out to already-active users of your app can be a powerful tool in helping to build a dedicated following of consistent app users. A little bit of personalized recognition of your power users can turn them into evangelists for your app.
 
-You can also check out our [LAB course](http://lab.braze.com/quick-overview-segment-and-campaign-setup) on marketing strategy for email and recommended lifecycle campaigns!
+You can also check out our [Braze Learning course](http://learning.braze.com/quick-overview-segment-and-campaign-setup) on marketing strategy for email and recommended lifecycle campaigns!
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/campaigns/ideas_and_strategies/active_user_campaigns.md:21]:
```suggestion
You can also check out our [Braze Learning course](https://learning.braze.com/quick-overview-segment-and-campaign-setup) on marketing strategy for email and recommended lifecycle campaigns!
```

File: _docs/_user_guide/engagement_tools/canvas.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ article_title: Canvas
...
+guide_top_text: "Marketing is a science — one that needs an artist’s touch and specialized tools. With Canvas, you can mix rigor and artistry to create meaningful, relevant, and personal experiences for each customer. <br> <br> Canvas is a single unified interface where marketers can set up campaigns with multiple messages and steps to form a cohesive journey and then compare and optimize those experiences using comprehensive analytics for the full user experience. <br> <br> The following articles will walk you through setting up a Canvas and up-level your strategies as you build customer experiences. You can also check out our <a href='http://learning.braze.com/canvas-course'>Canvas Braze Learning course</a>!"
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/canvas.md:7]:
```suggestion
guide_top_text: "Marketing is a science — one that needs an artist’s touch and specialized tools. With Canvas, you can mix rigor and artistry to create meaningful, relevant, and personal experiences for each customer. <br> <br> Canvas is a single unified interface where marketers can set up campaigns with multiple messages and steps to form a cohesive journey and then compare and optimize those experiences using comprehensive analytics for the full user experience. <br> <br> The following articles will walk you through setting up a Canvas and up-level your strategies as you build customer experiences. You can also check out our <a href='https://learning.braze.com/canvas-course'>Canvas Braze Learning course</a>!"
```

File: _docs/_user_guide/engagement_tools/canvas/create_a_canvas/create_a_canvas.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ tool: Canvas
 
 # Creating a Canvas
 
-> This reference article covers the necessary steps involved in creating, maintaining, and testing a Canvas. Follow this guide, or check out our [Canvas LAB course](http://lab.braze.com/quick-overview-canvas-setup)!
+> This reference article covers the necessary steps involved in creating, maintaining, and testing a Canvas. Follow this guide, or check out our [Canvas Braze Learning course](http://learning.braze.com/quick-overview-canvas-setup)!
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/canvas/create_a_canvas/create_a_canvas.md:13]:
```suggestion
> This reference article covers the necessary steps involved in creating, maintaining, and testing a Canvas. Follow this guide, or check out our [Canvas Braze Learning course](https://learning.braze.com/quick-overview-canvas-setup)!
```

File: _docs/_user_guide/data_and_analytics/custom_data/custom_events.md [hunk 5]
```diff
@@ -12,7 +12,7 @@ description: "This reference article describes custom events and properties, the
 Custom events are actions taken by, or updates about, your users. They're best suited for tracking high-value user interactions within your application. Logging a custom event can trigger any number and type of follow-up campaigns, and enables the listed segmentation filters on the recency and frequency of that event.
 
 {% alert tip %}
-For more on using custom events in your messaging strategies, check out our [Custom Events and Attributes](http://lab.braze.com/custom-events-and-attributes) LAB course!
+For more on using custom events in your messaging strategies, check out our [Custom Events and Attributes](http://learning.braze.com/custom-events-and-attributes) Braze Learning course!
```
Comment (lydia-xie) [inline _docs/_user_guide/data_and_analytics/custom_data/custom_events.md:15]:
```suggestion
For more on using custom events in your messaging strategies, check out our [Custom Events and Attributes](https://learning.braze.com/custom-events-and-attributes) Braze Learning course!
```

File: _docs/_user_guide/engagement_tools/campaigns/building_campaigns/conversion_events.md [hunk 6]
```diff
@@ -20,7 +20,7 @@ A conversion event is a type of success metric that tracks whether a recipient o
 
 For example, if you're creating a personalized holiday campaign for active users, a conversion event of **Start a Session** within two or three days may be appropriate since it will allow you to gather a sense of user engagement from receiving your message. Additional events like **Make Purchase**, **Upgrade App**, or any of your custom events can be selected as conversion events.
 
-For more on conversions, check out our [LAB course](http://lab.braze.com/campaign-setup-delivery-targeting-conversions) on campaign setup!
+For more on conversions, check out our [Braze Learning course](http://learning.braze.com/campaign-setup-delivery-targeting-conversions) on campaign setup!
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/campaigns/building_campaigns/conversion_events.md:23]:
```suggestion
For more on conversions, check out our [Braze Learning course](https://learning.braze.com/campaign-setup-delivery-targeting-conversions) on campaign setup!
```

**Review by lydia-xie — CHANGES_REQUESTED** [review]
looks great! just left some suggestions for your review (HTTP to HTTPS)

**Review by KellieHawks — CHANGES_REQUESTED** [review]
Looks great! Can you update this one too: /braze-docs/_layouts/doc_guide.html
Do you know if the Braze learning team plans to create a new logo?

**Review by KellieHawks — APPROVED** [review]

**Review by lydia-xie — APPROVED** [review]
lgtm!

**Pull request merged**